### PR TITLE
Upgrade to cvxpy 1.0

### PR DESCRIPTION
**Pull request by luthfianto**
Hi, I'm currently learning ncvx. Might as well making it compatible with cvxpy 1.0 first

### CODE REVIEW

File: ncvx/tests/test_vars.py [hunk 1]
```diff
@@ -32,40 +32,40 @@ def setUp(self):
 
     # Test boolean variable.
     def test_boolean(self):
-        x = Variable(5, 4)
+        x = Variable((5, 4))
         y = Boolean(5, 4)
-        p = Problem(Minimize(sum_entries(1-x) + sum_entries(x)), [x == y])
+        p = Problem(Minimize(sum(1-x) + sum(x)), [x == y])
```
Comment (SteveDiamond) [inline ncvx/tests/test_vars.py:38]: Make sure this sum is from cvxpy

**Review by SteveDiamond — COMMENTED** [review]
This looks good! One request is to make sure that when sum is used it's from cvxpy and not the built-in sum.

File: ncvx/tests/test_vars.py (hunk 1 above)
Comment (luthfianto) [reply ncvx/tests/test_vars.py:38]: Got it, I use `cp.sum()` now

**Comment by SteveDiamond** [comment]
Thanks for this super helpful PR!

**Pull request merged**